File: zxing-sample/src/main/java/me/dm7/barcodescanner/zxing/sample/SimpleScannerFragmentActivity.java
package me.dm7.barcodescanner.zxing.sample;

import android.os.Bundle;

import me.dm7.barcodescanner.zxing.sample.databinding.ActivitySimpleScannerFragmentBinding;

public class SimpleScannerFragmentActivity extends BaseScannerActivity {
    @Override
    public void onCreate(Bundle state) {
        super.onCreate(state);
        ActivitySimpleScannerFragmentBinding binding = ActivitySimpleScannerFragmentBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
        setupToolbar(binding.toolbar);
    }
}

File: zxing-sample/src/main/java/me/dm7/barcodescanner/zxing/sample/simplescanner/SimpleScannerFragmentActivity.kt
package me.dm7.barcodescanner.zxing.sample.simplescanner

import android.os.Bundle
import me.dm7.barcodescanner.zxing.sample.base.BaseScannerActivity
import me.dm7.barcodescanner.zxing.sample.databinding.ActivitySimpleScannerFragmentBinding

class SimpleScannerFragmentActivity : BaseScannerActivity() {

    override fun onCreate(state: Bundle?) {
        super.onCreate(state)

        val binding = ActivitySimpleScannerFragmentBinding.inflate(layoutInflater)
        setContentView(binding.root)

        setupToolbar(toolbar = binding.toolbar)
    }
}
